### PR TITLE
Christiaan/notification screen

### PR DESCRIPTION
**Pull request by WhoShotCupidd**

### CODE REVIEW

File: code/noRAM/app/src/main/java/com/example/noram/model/Notification.java
Comment (WhoShotCupidd) [inline code/noRAM/app/src/main/java/com/example/noram/model/Notification.java:1]: @ServingCoder

File: code/noRAM/app/src/main/java/com/example/noram/model/Event.java
Comment (WhoShotCupidd) [inline code/noRAM/app/src/main/java/com/example/noram/model/Event.java:1]: @ServingCoder here I add the list of notifications to the event the same can be done for you

File: code/noRAM/app/src/main/java/com/example/noram/model/Notification.java
Comment (WhoShotCupidd) [reply code/noRAM/app/src/main/java/com/example/noram/model/Notification.java:1]: here is the new class

File: code/noRAM/app/src/main/java/com/example/noram/model/PushNotificationService.java [hunk 1]
```diff
@@ -158,5 +162,17 @@ public void sendNotification(String title, String data, Event event, Boolean sen
...
     }
+
+    /**
+     * A method to send a notification to the attendees of an event
+     * @param title the title of the notification
+     * @param data the data of the notification
+     * @param event the event to send the notification to
+     */
+    private void sendNotificationToDB(String title, String data, Event event) {
+        event.addNotification(new Notification(title, data));
```
Comment (WhoShotCupidd) [inline code/noRAM/app/src/main/java/com/example/noram/model/PushNotificationService.java:173]: this is me adding it to the db @ServingCoder

File: code/noRAM/app/src/main/java/com/example/noram/model/Notification.java
Comment (ServingCoder) [reply code/noRAM/app/src/main/java/com/example/noram/model/Notification.java:1]: thanks

**Pull request merged**